release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2023,6 +2023,22 @@ As a workaround, use fully qualified names, such as `registry.redhat.io/ubi8/ubi
+
+* On an {product-title} single node configuration, pod creation times are over two times slower when using the real-time kernel (`kernel-rt`) than when using the non-real time kernel. When using `kernel-rt`, the slower pod creation times affect the maximum number of supported pods because recovery time is impacted after a node reboots.
++
+As a workaround, when you use `kernel-rt`, you can improve the recovery time by booting with the `rcupdate.rcu_normal_after_boot=0` kernel argument. This requires a real-time kernel version `kernel-rt-4.18.0-305.16.1.rt7.88.el8_4` or later. This known issue applies to {product-title} version 4.8.15 and later. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1975356[BZ#1975356])
+
+Following an {product-title} single node reboot, all pods are restarted which causes significant load and longer than normal pod creation times. This happens because the Container Network Interface (CNI) is not able to process the `pod add` events quickly enough. The following error message is displayed: `timed out waiting for OVS port binding`. The {product-title} single node instance eventually recovers, though more slowly than expected. This known issue applies to {product-title} version 4.8.15 and later. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986216[BZ#1986216])
+
+* An error occurs during SNO cluster provisioning where `bootkube` tries to use `oc` towards the end of the cluster bootstrap process. The kube API receives a shutdown request and this causes the cluster bootstrap process to fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2010665[BZ#2010665])
+
+* Deploying an {product-title} version 4.9 SNO cluster after a successful 4.8 deployment on the same host fails due to a modified boot table entry. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011306[BZ#2011306])
+
+* There is an instability issue with the inbox iavf driver that is evident when a DPDK-based workload is deployed in {product-title} version 4.8.5. The issue is also apparent when a DPDK workload is deployed on a host running RHEL for Real Time 8. The issue occurs in hosts with Intel XXV710 NICs installed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2000180[BZ#2000180])
+
+* A clock jump error occurs in the `linuxptp` subsystem that is deployed by the PTP Operator. The reported error message is: `clock jumped backward or running slower than expected!`. The error is encountered in a host with an Intel Columbiaville E810 NIC installed in a {product-title} version 4.8 or 4.9 cluster. The error is likely ice driver related, rather than an error in the `linuxptp` subsystem. An imminent Intel driver update and {product-title} version 4.9 support for the Columbiaville NIC will resolve this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2013478[BZ#2013478])
```

Comment:
```suggestion
* A clock jump error occurs in the `linuxptp` subsystem that is deployed by the PTP Operator. The reported error message is: `clock jumped backward or running slower than expected!`. The error is encountered in a host with an Intel Columbiaville E810 NIC installed in a {product-title} version 4.8 or 4.9 cluster. The error is likely ice driver related, rather than an error in the `linuxptp` subsystem. An imminent Intel ice driver update and {product-title} version 4.9 support for the Columbiaville NIC resolves this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2013478[BZ#2013478])
```

Reply:
Just for consistency adding ice would be nice :) also avoid future tense if possible.